tools/feht/feht.xml

```diff
@@ -0,0 +1,120 @@
+<tool id="feht" name="feht" version="0.1.0">
+    <description>A commandline program to automatically identify makers predictive of groups.</description>
+    <requirements>
+        <requirement type="package" version="1.1.0">feht</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        feht -i $metadata -d $data_file -m $binary_or_snp > output.txt
```

Comment:
single quotes around $metadata $data_file and $binary_or_snp

Reply:
too long man. Perhaps change it to " automatically identify makers predictive of groups."